pages/services/prometheus/0.1.0-2.3.1/security/index.md

```diff
@@ -0,0 +1,17 @@
+---
```

Comment:
If it doesnt apply, then why do you need the section?

Reply:
Removed